ui/core/individual_sim_ui.ts

```diff
@@ -393,20 +391,6 @@ export abstract class IndividualSimUI<SpecType extends Spec> extends SimUI {
 		//this.simHeader.addExportLink("CLI", _parent => new Exporters.IndividualCLIExporter(this.rootElem, this), true);
 	}
 
-	applyDefaultRotation(eventID: EventID) {
```

Comment:
This was an intentional add on my part to allow individual sims to specify the simple interface as what users see by default if desired. Should still work with the new code as long as the individual sim config supports optional fields within the defaults section for "rotationType?" and "simpleRotation?". I'm happy to re-commit it separately though if that's easier.

Reply:
The feral sim still shows the simple rotation and changing options does affect results, too.

<img width="1210" alt="image" src="https://github.com/wowsims/wotlk/assets/12898988/f3bf949a-8df9-4862-b168-c775db4d69d6">

Reply:
And with full buffs I get ~306 dps

<img width="273" alt="image" src="https://github.com/wowsims/wotlk/assets/12898988/a74b855e-a23a-4569-9e0c-fba92910f7d2">

Reply:
Yeah that commit wouldn't change results at all, just which interface is shown when loading defaults.